fiftyone/utils/open_clip.py

```diff
@@ -0,0 +1,122 @@
+"""
+CLIP model wrapper for the FiftyOne Model Zoo.
+
+| Copyright 2017-2023, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import logging
+import os
+from packaging.version import Version
+import warnings
+
+import eta.core.web as etaw
+
+import fiftyone as fo
+import fiftyone.core.models as fom
+import fiftyone.core.utils as fou
+import fiftyone.utils.torch as fout
+import fiftyone.zoo.models as fozm
+
+fou.ensure_torch()
+import torch
+
+import open_clip
```

Comment:
Can you use this style of importing instead?

```py
open_clip = fou.lazy_import(
    "open_clip",
    callback=lambda: fou.ensure_package("open_clip_torch")
)
```

This allows for the module to be imported even if this (non-default) dependency is not available, and then raises an informative error message only at runtime telling the user they need to install `open_clip_torch` in order to use this feature.